### PR TITLE
Use pip yolox instead of clone

### PR DESCRIPTION
**Pull request by yangw1234**
## Why this change?
Use pip yolox instead of clone

## What does this PR include?
1. remove download_models.sh
2. use yolox==0.3.0 instead of cloning from source
3. download weight file when setting up detection (if file not already exists)
4. update README.md installation instructions.

## User API changes
new installation steps:

1. setup environment
```shell
conda create -n vqpy python=3.8  # "vqpy" is conda environment name, you can use any name you like.
conda activate vqpy
pip install torch torchvision numpy==1.23.5 cython
```

5. install vqpy
```shell
git clone https://github.com/vqpy/vqpy.git
cd vqpy
pip install .
```

## How did I test the PR?
existing unit tests

## New dependencies included
- [x] I have added the dependencies in `setup.py`, yolox==0.3.0

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,5 +33,9 @@ def get_description():
                       "numpy<1.24.0",
                       "requests",
                       "opencv-python",
+                      "yolox==0.3.0",
+                      "cython_bbox",
+                      "lap",
+                      "openalpr==1.0"
```
Comment (yushan111) [inline setup.py:39]: Add option of vehicle in `extra_requires`

**Comment by yushan111** [comment]
Could you also help test the installation on colab to ensure it works?

**Comment by yangw1234** [comment]
> Could you also help test the installation on collab to ensure it works without conda environment?

`pip install -e .` does not work in colab. Changed to `pip install .`

File: setup.py (hunk 1 above)
Comment (yushan111) [reply setup.py:39]: leave it in future

**Comment by yushan111** [comment]
Thanks for the great job!

**Pull request merged**